### PR TITLE
[ai-did-you-mean-this] Rename Required Service Parameter "sessionId" to "correlationId"

### PR DESCRIPTION
**Pull request by christopher-o-toole**
Rename required service parameter "sessionId" to "correlationId". Required by pending changes to the Aladdin prod endpoint.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
ai-did-you-mean-this

File: src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 # Commands
 # note: at least one command is required in order for the CLI to load the extension.
 def show_extension_version():
-    print(f'Current version: 0.1.1')
+    print(f'Current version: 0.2.0')
```
Comment (haroldrandom) [inline src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py:35]: If there is no placeholder, then there is no need to use f-string.
```suggestion
    print('Current version: 0.2.0')
```

File: src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py [hunk 2]
```diff
@@ -207,7 +207,7 @@ def call_aladdin_service(command, parameters, version):
 
     if subscription_id and correlation_id:
         context = {
-            "sessionId": correlation_id,
+            "correlationId": correlation_id,
```
Comment (fengzhou-msft) [inline src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py:210]: How do you deal with the requests sent by older extensions that still use `sessionId`?

File: src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py (hunk 1 above)
Comment (christopher-o-toole) [reply src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py]: Removed use of f-string.

File: src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py (hunk 2 above)
Comment (christopher-o-toole) [reply src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py:210]: The service that was just deployed today supports both the `sessionId` parameter and the `correlationId` parameter.

**Pull request merged**